inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -80,6 +80,8 @@ uint32_t GNADeviceHelper::propagate(const uint32_t requestConfigId, Gna2Accelera
     checkGna2Status(status1);
     const auto status2 = Gna2RequestEnqueue(requestConfigId, &reqId);
     checkGna2Status(status2);
+    wasAnyRequestScheduled = true;
```

Comment:
consider to falsify this flag if gna2requestwait was called and got 'full completition' status.

Reply:
it is indeed a mistake, it was supposed to be false